go.mod

```diff
@@ -44,3 +43,5 @@ require (
 	gopkg.in/yaml.v2 v2.3.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 )
+
+replace github.com/mitchellh/gox => github.com/skycoin/gox v1.0.1
```

Comment:
Should we just change the import names directly in the source files?

Reply:
But this dependency is not even used?

Reply:
Maybe leftover and not removed yet? Also, I do not quite get why it would even be a dependency. It needs to be manually installed by the user who wants to cross compile afair.